app/mailers/emailer.rb

```diff
@@ -12,24 +12,5 @@ def appreciation_email(appreciation)
     mail(from: default_from_email,
          to: @recipient.email,
          subject: "We received an appreciation for #{@recipient.name} on #{Date.current}!! Yippee!")
-  end
-
-  def feedback_email(feedback)
-    @recipient = feedback.recipient
-    @url = feedback_url(feedback)
-    @message = feedback
-    mail(from: default_from_email,
-         to: @recipient.email,
-         subject: "New feedback for #{@recipient.name} on #{Date.current}!! Excelsior!")
-  end
-
-  def comment_email(comment)
-    @feedback = comment.root
-    @url = send(:"#{@feedback.class.name.downcase}_url", comment.root)
-    @message = comment
-    mail(from: default_from_email,
-         to: @message.recipient.email,
-         subject: "New reply to feedback for #{@feedback.recipient.name} on #{Date.current}!! Discuss!")
-  end
-
+    end
```

Comment:
[pronto-rubocop warning] Layout/DefEndAlignment: `end` at 15, 4 is not aligned with `def` at 8, 2.

Reply:
Fixing